### PR TITLE
fix FPU CPACR bitmasks

### PR DESCRIPTION
**Pull request by sbourdeauducq**
See http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0553a/BABDBFBJ.html

### CODE REVIEW

File: src/peripheral/mod.rs [hunk 1]
```diff
@@ -443,9 +443,9 @@ pub enum FpuAccessMode {
     Privileged,
 }
 
-const SCB_CPACR_FPU_MASK: u32 = 0x00780000;
-const SCB_CPACR_FPU_ENABLE: u32 = 0x00280000;
-const SCB_CPACR_FPU_USER: u32 = 0x00500000;
+const SCB_CPACR_FPU_MASK: u32 = 0x00f00000;
```
Comment (japaric) [inline src/peripheral/mod.rs]: Could you rewrite these as  `0b1111 << 20`? I think that format would increase readability and avoid off by one errors.

Comment (sbourdeauducq) [inline src/peripheral/mod.rs]: Done

**Comment by japaric** [comment]
Thank you @sbourdeauducq.

@homunkulus r+

**Comment by homunkulus** [comment]
:pushpin: Commit 3951582 has been approved by `japaric`

**Comment by homunkulus** [comment]
:hourglass: Testing commit 3951582 with merge 3951582...

**Comment by homunkulus** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/japaric/cortex-m/builds/241885271?utm_source=github_status&utm_medium=notification)
Approved by: japaric
Pushing 39515828cae8a3e83d202d5383053d0efe8cfa10 to master...

**Pull request merged**